lib/configuration/client_configuration.js

```diff
@@ -198,20 +198,20 @@ function ConfigurationUtil(fsPromisesModule, processModule) {
 
   async function findConfig(filePathFromConnectionString) {
     if (exists(filePathFromConnectionString)) {
-      Logger.getInstance().info('Using client configuration path from a connection string: %s', filePathFromConnectionString);
+      Logger.getInstance().debug('Using client configuration path from a connection string: %s', filePathFromConnectionString);
```

Comment:
The logging of the configuration is required to be kept on the .info level due to the risk mitigations requested by the security.